vagrant/config/versions.yaml

```diff
@@ -68,7 +68,8 @@ installers:
 
   - foreman: '2.5'
     katello: '4.1'
-    pulpcore: '3.11'
+    pulp: '2.21'
```

Comment:
This looks like a bad rebase that ends up reverting ab476516d1cf9809d9bc02c6456bdbebbf40c2df
```suggestion
```